package.json

```diff
@@ -27,25 +27,52 @@
   "eslintConfig": {
     "root": true,
     "env": {
-      "node": true
+      "browser": true,
+      "jquery": true
     },
     "extends": [
-      "plugin:vue/essential",
-      "eslint:recommended"
+      "eslint:recommended",
+      "plugin:vue/recommended"
+    ],
+    "globals": {
+      "CONFIG": "readonly",
+      "CONST": "readonly",
+      "Actor": "readonly",
+      "Application": "readonly",
+      "ContextMenu": "readonly",
+      "Dialog": "readonly",
+      "Dlopen": "readonly",
+      "FilePicker": "readonly",
+      "Folder": "readonly",
+      "ForgeCompendiumBrowser": "readonly",
+      "ForgeCompendiumBrowserVue": "readonly",
+      "ForgeVTT": "readonly",
+      "FormDataExtended": "readonly",
+      "Hooks": "readonly",
+      "ImageHelper": "readonly",
+      "ImagePopout": "readonly",
+      "Item": "readonly",
+      "JournalEntry": "readonly",
+      "JournalSheet": "readonly",
+      "Scene": "readonly",
+      "Tabs": "readonly",
+      "Vue": "readonly",
+      "canvas": "readonly",
+      "dndbeyond_br": "readonly",
+      "foundry": "readonly",
+      "game": "readonly",
+      "getDocumentClass": "readonly",
+      "globalThis": "readonly",
+      "renderTemplate": "readonly",
+      "ui": "readonly"
+    },
```

Comment:
Some of these variables might be better scoped if we call it from [the `client` class](https://foundryvtt.com/api/modules/client.html). I will look into it once I am able to test the module properly.

I cannot find declaration or documentation for `dndbeyond_br`, `getDocumentClass` and `renderTemplate`: where is it coming from?